README.md

```diff
@@ -35,6 +33,29 @@ Configuration options available at `admin/islandora/tools/islandora_usage_stats`
 * Does not respect XACML.
 * This is a server-side tracking solution, as such a caching layer could impact it.  If this is impacting you a [solution](https://github.com/discoverygarden/islandora_ga_reports) using JavaScript may work better.
 
+## Troubleshooting/Issues
+
+Having problems or solved a problem? Check out the Islandora google groups for a solution.
+
+* [Islandora Group](https://groups.google.com/forum/?hl=en&fromgroups#!forum/islandora)
+* [Islandora Dev Group](https://groups.google.com/forum/?hl=en&fromgroups#!forum/islandora-dev)
+
+## Maintainers/Sponsors
+
+Work based on code from https://github.com/roblib/islandora_scholar_upei and the scholar_tracking module for Drupal 6. Iterated on by the Ryerson Digital Repository and discoverygarden inc.
```

Comment:
Can we change:

-  Ryerson Digital Repository
+ Ryerson University Library and Archives (RULA)